dom/events.ts

```diff
@@ -29,11 +29,6 @@ type GenericEventListener<T extends Event = Event> = {
  */
 export function on<T extends Event = Event> (element : null|EventTarget|EventTarget[], type : string|string[], handler : GenericEventListener<T>) : void
```

Comment:
We should introduce a special type for the EventTarget (you can't name it like this, as it already exists).

That type should be:

```typescript
type ohai = null | EventTarget | (null|EventTarget)[];
```